library/alloc/src/sync.rs

```diff
@@ -69,42 +69,42 @@ macro_rules! acquire {
 /// A thread-safe reference-counting pointer. 'Arc' stands for 'Atomically
 /// Reference Counted'.
 ///
-/// The type `Arc<T>` provides shared ownership of a value of type `T`,
-/// allocated in the heap. Invoking [`clone`][clone] on `Arc` produces
-/// a new `Arc` instance, which points to the same allocation on the heap as the
-/// source `Arc`, while increasing a reference count. When the last `Arc`
+/// The type [`Arc`]`<T>` provides shared ownership of a value of type `T`,
```

Comment:
This will currently render quite badly: https://github.com/rust-lang/rust/issues/62867

Reply:
Ah right. We can mitigate this problem by adding this CSS:

```css
.docblock code + a > code {
    padding: 0;
}
```

But it doesn't fix the right padding on the link. Still better than nothing though I guess.

Reply:
The link would be on the whole `Arc<T>`. I replicated what was there originally. It's done like that in a lot of places.